source/_docs/unsupported-modules-plugins.md

```diff
@@ -277,6 +277,14 @@ For more details, see [SERVER_NAME and SERVER_PORT on Pantheon](/docs/server_nam
 **Issue**: The Instashow plugin relies on query parameters that are not compatible with Pantheon's Edge Cache. See [PANTHEON_STRIPPED](https://pantheon.io/docs/pantheon_stripped/) for more information. This inhibits the ability to set the authorization token required to make the plugin function.
 <hr>
 
+### [TubePress](https://wordpress.org/plugins/tubepress/)
+**Issue**: TubePress will not work if you are using PHP 5. The server requirements for TubePress state that the PHP APC extension should be disabled. Pantheon provides APC by default across all plans and it is not possible to disable.
+
+**Solution**: Upgrade your site's PHP version to PHP 7 by enabling that option in a pantheon.yml file.
+
+See [Upgrade PHP Versions](https://pantheon.io/docs/php-versions/) and [The pantheon.yml Configuration File](https://pantheon.io/docs/pantheon-yml/) for more information.
+<hr>
```

Comment:
@twooten Is the solution based on the assumption that the APC extension is not enabled on PHP 7.0? Looks like the platform enables APC on all PHP versions: 5.3, 5.5, 5.6, and 7.0.

Here's some output from `phpinfo();` on a WP site running 7.0: 
![screen shot 2017-01-23 at 12 15 07 pm](https://cloud.githubusercontent.com/assets/10119525/22216776/03067696-e166-11e6-8e13-08efbcdfccf2.png)